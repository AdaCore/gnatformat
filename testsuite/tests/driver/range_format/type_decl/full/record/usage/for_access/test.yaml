description: ObjectDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 95",
    "-EL 96",
    "--pipe",
  ]
