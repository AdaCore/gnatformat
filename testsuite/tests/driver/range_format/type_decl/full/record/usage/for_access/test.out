test.ads(<ObjectDecl ["My_Car", "Your_Car", "Next_Car"] test.ads:95:4-95:53>) - 95:1-95:53
^
   My_Car, Your_Car, Next_Car : Car_Name := new Car;$

